build-packages/eslint-plugin-scandipwa-guidelines/lib/util/eslint-traverse.js

```diff
@@ -0,0 +1,60 @@
+// based on https://github.com/discord/eslint-traverse/blob/master/index.js
+// modification: use evk.KEYS instead of context.getSourceCode().visitorKeys, which doesn't appear to work when testing
+const evk = require("eslint-visitor-keys");
+
+"use strict"
+
+const SKIP = Symbol("skip")
+const STOP = Symbol("stop")
+
+function traverse(node, visitor) {
+    let allVisitorKeys = evk.KEYS
```

Comment:
This is the only change I'm unsure about.

This file is copied from the [original](https://github.com/discord/eslint-traverse/blob/master/index.js) which we were using previously -- but when I ran the unit tests,  `context.getSourceCode().visitorKeys` was for some reason always undefined (bug in eslint? idk. found no references to this online).

So I changed `context.getSourceCode().visitorKeys` to `evk.KEYS` (I'm pretty sure they're supposed to be equivalent). Now the code passes the unit tests.

I'm open to improvement suggestions, but I don't think having to copy this 60-line file is terrible.